app/controllers/api/github_issues_controller.rb

```diff
@@ -3,11 +3,13 @@ class Api::GithubIssuesController < ApplicationController
   before_action :create_github_client, only: [:index, :create]
 
   def index
-    @issues = @client.issues.list
+    @issues = @client.issues.list user: 'TeamMartinez', repo: 'YOLO-api', state: 'open'
     render json: @issues
   end
 
   def create
+    # TODO: make user: and repo: params default somewhere and use params
+    @client.issues.create user: 'TeamMartinez', repo: 'YOLO-api', title: params[:title]
     render json: { errors: "not implemented" }
```

Comment:
Two things

1. `require` can be chained with a `.permit` to encmpass all - And yes they are [rails specific](http://api.rubyonrails.org/classes/ActionController/Parameters.html)
2. What we want is consistent parameters for `user: 'TeamMartinez', repo: 'YOLO-api'`. That doesen't seem like something we can do with `params.require / permit`